From cc7892278238c6738fb492c2c509c83feb8e5f35 Mon Sep 17 00:00:00 2001
Message-Id: <cc7892278238c6738fb492c2c509c83feb8e5f35.1398812351.git.mark.gross@intel.com>
In-Reply-To: <e87e3533536ae5eec0e06ceb7ef7428e42855790.1398812350.git.mark.gross@intel.com>
References: <e87e3533536ae5eec0e06ceb7ef7428e42855790.1398812350.git.mark.gross@intel.com>
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Wed, 11 Dec 2013 12:37:21 +0000
Subject: [PATCH 208/429] ACPI: platform: Add ACPI platform device for WPT
 Audio DSP

Add INT3438 for Wild Cat Point audio DSP.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
---
 drivers/acpi/acpi_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index dbfe49e..61e66e2 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -35,6 +35,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 
 	/* Intel Smart Sound Technology */
 	{ "INT33C8" },
+	{ "INT3438" },
 	{ "80860F28" },
 
 	{ }
-- 
1.8.3.2

